core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/package.scala

```diff
@@ -4,4 +4,13 @@ package object leonardo {
   type LabelMap = Map[String, String]
   //this value is the default for autopause, if none is specified. An autopauseThreshold of 0 indicates no autopause
   final val autoPauseOffValue = 0
+
+  /** Aliases */
```

Comment:
what's the purpose of these aliases?

Reply:
I didn't update `Cluster` -> `Runtime`, `ClusterName` -> `RuntimeName` in all places… just to make refactoring easier.

Reply:
ah..can you add a comment about this? in a separate PR is fine too

Reply:
Will do